src/routes/blog/post/appwrite-decoded-sara-kaandorp/+page.markdoc

```diff
@@ -0,0 +1,50 @@
+category: culture
+---
+
+At Appwrite, we value our people, the hard work they do, and the culture they create. Appwrite Decoded is an opportunity to get to know the Appwrite team just a little bit better and for us to celebrate the person behind the work.
+
+We kick off our very first of this series with someone who has led Appwrite through many milestones, our very talented Design Lead, Sara Kaandorp.
+
+# From UX/UI to visual identity
+
+Sara Kaandorp helps developers build like a team of hundreds by leading a successful design team with many talents. In this blog, we put Sara in the spotlight and share her side of the story. 
+
+Sara's family background, half Dutch, half Finnish, is not the only duality she has. Throughout her career, she has worked as a UX Designer in diverse industries, where she has gained experience and skills in both visual and product design. As she leads a team of four, she plays a central role in shaping Appwrite's overall look and feel from both a visual brand as well as the UX/UI of the product; a conscious decision to ensure that both product and visual design work closely together on a cohesive experience.
+
+When it comes to her favorite aspects of her job, Sara emphasizes the dynamic nature of each day, highlighting the opportunities to work on diverse projects and aspects consistently. She gets to experiment with different design languages, continuously pushing boundaries with her team to ensure a better developer experience. Sara appreciates the chance for growth on both personal and team levels, highlighting the freedom of work that contributes to maintaining her excitement throughout her professional life. "Designing for developers has been a unique and challenging experience; they tend to be critical and honest; however, I'm personally grateful for this, as it has genuinely challenged our team to continually improve our standards. When we create something exceptional, they express their appreciation vocally, and that's very rewarding."
+
+# Team dynamics and building like a team of hundreds
+
+When it comes to being part of a team, and in Sara’s case, leading one, the key point is to unite people with diverse qualities rather than relying on those with similar qualifications. In her view, everyone has unique and impressive qualities, but no one has it all. Being a team involves supporting each other to cover all bases as a team, filling each other’s weaknesses, and, most importantly, placing complete trust in everyone throughout the process. "I always try to match people with different qualities, and of course, what matches with the objectives of Appwrite."
+
+Sara appreciates the fact that they have created an amazing team comprising successful, young, and passionate people who put considerable effort and creativity into their work. She consistently prioritizes keeping her team members personally motivated by maintaining a diverse workflow to prevent falling into the monotony of the same routine.
```

Comment:
```suggestion
Sara appreciates the fact that they have created an amazing team comprising successful, young, and passionate people who put considerable effort and creativity into their work. She consistently prioritizes keeping her team members motivated by maintaining a diverse workflow to prevent falling into the monotony of the same routine.
```